apps/pxweb2/public/locales/en/translation.json

```diff
@@ -81,7 +81,7 @@
         }
       },
       "view": {
-        "title": "View",
+        "title": "Distplay",
```

Comment:
Not correct english

Reply:
Is it "View" or "Display" that is incorrect?